### ISSUE TITLE
Prefetch not working if there is a Service Worker registered

### ISSUE THREAD

**Issue by robertgr991** [WICG/nav-speculation#317]
While trying to implement Speculative loads, I encountered the failure "The URL was not eligible to be prefetched because there was a registered service worker for that origin, which is currently not supported.". The prefetch urls are only "same-site". The Service Worker is used to cache fonts, JS and CSS resources. If the worker is disabled, speculations works as expected.

I've tried to find information on this error but couldn't find anything. `<link rel="prefetch">` doesn't cause this issue.

Does anyone have any input on this?

![Screenshot 2024-05-15 113847](https://github.com/WICG/nav-speculation/assets/36689800/7a5faf11-6512-41c8-81aa-c3427271c8c7)

**Comment by jeremyroman** [comment]
Yes, this is currently a shortcoming of the Chromium implementation and I believe also the spec, one which we should fix.

@hiroshige-g I believe this corresponds to [this Chromium bug](https://crbug.com/40947546) and [this document](https://docs.google.com/document/d/12_G2S0hI-G54f2c0ZOLtPq_fXZRjWvibGZfndIia1J4/edit). Is anything else required to track this?

**Comment by domenic** [comment]
Issue triage: let me close this, as this is mostly a Chrome-specific issue. Although there are several open spec issues about edge cases when prefetch and service worker interact, which we will separately keep open.
